code/modules/clothing/head/misc.dm

```diff
@@ -502,4 +502,40 @@
 	desc = "A marine helmet prop from the popular game 'Ruin'."
 	icon_state = "marine"
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|BLOCKHAIR
-	body_parts_covered = HEAD|FACE|EYES
\ No newline at end of file
+	body_parts_covered = HEAD|FACE|EYES
+
+/obj/item/clothing/head/ranger

```

Comment:
head/hardhat also sets
```
brightness_on = 4
armor = list(
	melee = 30,
	bullet = 5,
	laser = 20,
	energy = 10,
	bomb = 20,
	bio = 10,
	rad = 20
)
action_button_name = "Toggle Head-light"
w_class = ITEMSIZE_NORMAL
ear_protection = 1
```

Reply:
Actually mostly deliberate, this is a costume item, so does it really need _any_ gizmos/armor?